docker-compose.yaml

```diff
@@ -257,6 +257,29 @@ services:
       timeout: 10s
       retries: 3
       start_period: 40s
+  waypoint:
+    image: ${REGISTRY:-local}/waypoint:${IMAGE_TAG:-latest}
+    container_name: waypoint
+    build:
+      context: .
+      dockerfile: ./dockerfiles/waypoint/Dockerfile
+    ports:
+      - 0.0.0.0:3011:3011
+    volumes:
+      - ./waypoint:/waypoint
+      - ./logs:/logs
+    networks:
+      - governance-multitenant
+      - host
+      - nats
+    depends_on:
+      - governance-endorser
+    # depends_on:
+    #   - nats-1
+    #   - nats-2
+    #   - nats-3
+    env_file:
+      - environments/waypoint/waypoint.env
```

Comment:
Lastly we'd want to configure the healthcheck here.

Similar to how it's done for webhooks/endorser:
```py
    healthcheck:
      test: ["CMD-SHELL", "curl -f http://localhost:3010/health || exit 1"]
      interval: 30s
      timeout: 10s
      retries: 3
      start_period: 40s
```